Makefile

```diff
@@ -30,29 +30,17 @@ image_pip_list:
 e2e_neuro_push:
 	neuro push $(TARGET_IMAGE_NAME):built-$(BASE_IMAGE_TYPE) $(TEST_IMAGE_NAME):$(BASE_IMAGE_TYPE)
 
-TEST_PRESET=cpu-small
-TEST_CMD=
-.PHONY: _test_e2e
-_test_e2e:
+TEST_PRESET=gpu-small
```

Comment:
AFAIK we don't have GPUs in dev, so this will most likely fail

Reply:
we're testing it against staging's `default`, there we have `gpu-small` preset.
as for me, that preset is more likely to appear in the cluster than k80 :)

Reply:
ah, I see. missed the default cluster part. thanks!